tests/scripts/all.sh

```diff
@@ -4339,6 +4339,87 @@ component_build_aes_aesce_armcc () {
     armc6_build_test "-O1 --target=aarch64-arm-none-eabi -march=armv8-a+crypto"
 }
 
+support_build_sha_armce() {
+    if command -v clang > /dev/null ; then
+        # clang >= 4 is required to build with SHA extensions
+        clang_ver="$(clang --version|grep version|sed -E 's#.*version ([0-9]+).*#\1#')"
+
+        # we need asm/hwcap.h available for runtime detection
+        echo '#include <asm/hwcap.h>' | clang -E - >/dev/null 2>&1 && have_hwcap=1 || have_hwcap=0
```

Comment:
To fix CI failure, I guess we should not check it here and include `asm/hwcap.h` . Just define the bits when it is not defined. like bellow
```
#include <sys/auxv.h> //This is always available
#if !defined(HWCAP_SHA2)
#define HWCAP_SHA2
#endif
```

When `asm/hwcap.h` is available, some feature set bits are not available in old linux versions, those bits should be defined also.

Reply:
The CI fix is coming separately. I think @tom-cosgrove-arm had the same view on defining these constants? If so I'll add this